src/goto-programs/goto_program_template.h

```diff
@@ -80,60 +79,129 @@ class goto_program_templatet
     return *this;
   }
 
-  /*! \brief Container for an instruction of the goto-program
-  */
+  /// This class represents a node in the control flow graph of the function.
+  /// The successor nodes are determined by:
```

Comment:
I don't think the description of `instructiont` should be discussion "successors", see above -- it's not the right concept.